go.mod

```diff
@@ -4,7 +4,10 @@ go 1.14
 
 require (
 	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/felixge/httpsnoop v1.0.1 // indirect
+	github.com/gorilla/mux v1.8.0 // indirect
 	github.com/kr/pretty v0.1.0 // indirect
+	go.opentelemetry.io/contrib/propagators v0.12.0 // indirect
```

Comment:
These look like they were inadvertent. The `otelmux` module should be the only place these dependencies need to be added.

Reply:
yep, I failed to run `make precommit` 👍